auto_round/export/export_to_autoround/export.py

```diff
@@ -83,7 +82,8 @@ def dynamic_import_quant_linear_for_packing(backend, bits, group_size, sym, act_
         from ..export_to_awq.utils import WQLinear_GEMM
         return WQLinear_GEMM
     elif "gptqmodel" in backend:
-        return auto_round_extension.cuda.qlinear_tritonv2.QuantLinear
+        from gptqmodel.nn_modules.qlinear.torch import TorchQuantLinear # pylint: disable=E0401
```

Comment:
You'll need to port the packing code to our side. For the tuning process, it's best not to rely on external libraries.